src/lib/common.ts

```diff
@@ -46,7 +47,7 @@ export function createClassValidator(
 ): ClassValidator {
   const ajv = createValidator();
   for (const [k, v] of Object.entries(schema.definitions)) {
-    ajv.addSchema(v, `#/definitions/${k}`);
+    ajv.addSchema(v, `#/definitions/${url.format(k)}`);
```

Comment:
I'm pretty sure this is not a safe way to escape weird TypeScript characters inside a weirder [JSON pointer](https://tools.ietf.org/html/rfc6901).  Perhaps https://www.npmjs.com/package/json-pointer-rfc6901#escapestr---str or a similar method would be more reliable?
Failing that, please name things to make it clear what's going on, e.g.
```ts
const escapeJsonPointer = url.format;
```

Reply:
Addressed.